hadoop-hdds/docs/content/interface/Cli.md

```diff
@@ -165,6 +185,24 @@ $ ozone sh bucket info /vol1/bucket1
 }
 ```
 
+If the bucket is empty we can delete bucket using below command.
+
+```shell
+$ ozone sh bucket delete /vol1/bucket1
+Bucket bucket1 is deleted
+```
+
+If the bucket contains some keys, we can delete bucket recursively. Which will delete all the keys for the specified bucket and also bucket will be deleted.
```

Comment:
```suggestion
If the bucket contains any keys, we can delete the bucket recursively. This will delete all the keys within the bucket, and then the bucket itself.
```